dexlib2/src/main/java/org/jf/dexlib2/util/Preconditions.java

```diff
@@ -221,7 +221,7 @@ public static int checkArrayPayloadElementWidth(int elementWidth) {
 
     public static <L extends List<? extends Number>> L checkArrayPayloadElements(int elementWidth, L elements) {
         // mask of all bits that do not fit into an 'elementWidth'-bit number
```

Comment:
It should be `an 'elementWidth'-byte number`.

Reply:
Done